### PR TITLE
Fix toast issues 

### LINKED ISSUE #53: Toast "transaction succeeded" should include etherscan link

### LINKED ISSUE #54: Toast messages should not cover up wallet connection, should be dismissable

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Move toast to bottom left
- Make success toast dismissable
- Add link to success toast

Closes #53 
Closes #54

### CODE REVIEW

File: hooks/useNotifications.tsx [hunk 1]
```diff
@@ -13,25 +13,28 @@ import { useRef } from "react";
...
 
 const toastDefaultArgs = {
-  position: "top-right" as ToastPosition,
+  position: "bottom-right" as ToastPosition,
   duration: 5000,
   isClosable: true,
 };
 
+const ToastLink = ({ href, ...props }: ToastProps & { href: string }) => (
+  <Link href={href} isExternal>
```
Comment (wildmolasses) [inline hooks/useNotifications.tsx:22]: because the Link wraps the whole toast, a "dismiss" click takes us to the etherscan link.

**Review by wildmolasses — CHANGES_REQUESTED** [review]

File: hooks/useNotifications.tsx (hunk 1 above)
Comment (alexkeating) [reply hooks/useNotifications.tsx]: Thanks! Wrapped the description with the link instead and remove the whole custom toast link component

Comment (alexkeating) [reply hooks/useNotifications.tsx]: Also, removed closability of the pending toast  as that prevents the success toast from showing

**Review by wildmolasses — APPROVED** [review]
lgtm!

**Pull request merged**